packages/ia-components/sandbox/iajs-test-component/index.jsx

```diff
@@ -32,17 +32,18 @@ export default class extends Component {
   fetchMetadata(identifier) {
     let metadataService = new MetadataService()
     // TODO-ERRORS Note this has an unhandled error if MetadataService.get fails
-    metadataService.get({identifier: identifier}).then(metadata => {
-      this.setState({metadata: metadata.data})
+    metadataService.get({ identifier: identifier }).then(metadata => {
+      this.setState({ metadata: metadata.data })
     })
   }
 
-  render () {
+  render() {
     return (
       <div className="iajs-test-component">
         <h5>My identifier is {this.props.identifier}</h5>
         <img
-          style={{maxWidth: '300px', height: 'auto'}}
+          aria-label="iajs-test-component"
```

Comment:
This image is entirely decorative, and as such it makes more sense to simply add an empty `alt` attribute to conform to a11y standards as described in https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
The alt attribute should be empty, not populated with the parent div's class value.